app/script/event/EventRepository.coffee

```diff
@@ -35,7 +35,7 @@ class z.event.EventRepository
   @param user_repository [z.user.UserRepository] Repository for all user and connection interactions
   ###
   constructor: (@web_socket_service, @notification_service, @cryptography_repository, @user_repository) ->
-    @logger = new z.util.Logger 'z.event.EventRepository', z.config.LOGGER.OPTIONS
+    @logger = Logdown {alignOutput: true, prefix: 'z.event.EventRepository'}
 
```

Comment:
Logdown is the logger of choice now. All our log statements will soon be upgraded to Logdown (but in a separate PR).